### ISSUE TITLE
Plymouth doesn't work when running in Virtualbox (Bugzilla Bug 1582)

### ISSUE THREAD

**Issue by tpgxyz** [OpenMandrivaAssociation/test2#1582]
This issue was created automatically with bugzilla2github
# Bugzilla Bug 1582
Date: 2016-03-30 10:54:22 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;
To:   OpenMandriva QA &lt;<bugs@openmandriva.org>&gt;
CC:   agoiza@gmail.com, @benbullard79, @berolinux, @itchka, @jclvanier, @tapwag, @rugyada, @tpgxyz

Depends on:   #1583
Last updated: 2018-08-20 18:54:27 +0000

## Comment 12461
Date: 2016-03-30 10:54:22 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

for a long time, iso after iso plymouth doesn't work on my pc.

during boot I have grey screen with little squares , some with question marks (?) inside.

and it seems I am not the only one.

## Comment 12465
Date: 2016-03-30 11:12:59 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

working great here ....ahahahahhah

## Comment 12469
Date: 2016-03-30 11:20:39 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

seriously, I appreciate your effort

## Comment 12472
Date: 2016-03-30 11:46:04 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to luca from comment #2)
> seriously, I appreciate your effort

what's weird is that it works during installation, then the first boot and not any more after

## Comment 12474
Date: 2016-03-30 11:51:41 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

https://issues.openmandriva.org/show_bug.cgi?id=1289#c9

here work one time on ten.
so it's not a vbox issue, but probably a confict of booting services at start

## Comment 12476
Date: 2016-03-30 11:54:15 +0000
From: @tpgxyz

Can you run this as root on your installed Lx 3:
plymouth-set-default-theme -R OpenMandriva

and after it finishes, reboot and check splash screen.

## Comment 12481
Date: 2016-03-30 12:59:14 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

I said that sometimes plymouth splash appears, so it should be present in initrd.

anyway no changes

mostly it appears on exit, than on boot

## Comment 12487
Date: 2016-03-30 20:19:42 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to Tomasz Paweł Gajc from comment #5)
> Can you run this as root on your installed Lx 3:
> plymouth-set-default-theme -R OpenMandriva
> 
> and after it finishes, reboot and check splash screen.

I don't run your command, but now plymouth works (6 times)

what I did ?
install kernel (with panic)
remove this kernel bug #1583 
install updates like
kernel-firmware-20160330-1-omv2015.0.noarch
kernel-firmware-extra-20160330-1-omv2015.0.noarch

I understand nothing :)

to be continued ....

## Comment 12488
Date: 2016-03-30 20:30:17 +0000
From: @tpgxyz

(In reply to Joel Tesson from comment #7)

> kernel-firmware-20160330-1-omv2015.0.noarch
> kernel-firmware-extra-20160330-1-omv2015.0.noarch
> 
> I understand nothing :)

What gfx card do you have ? Maybe some firmware is missing

## Comment 12489
Date: 2016-03-30 20:32:04 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to Tomasz Paweł Gajc from comment #8)
> (In reply to Joel Tesson from comment #7)
> 
> > kernel-firmware-20160330-1-omv2015.0.noarch
> > kernel-firmware-extra-20160330-1-omv2015.0.noarch
> > 
> > I understand nothing :)
> 
> What gfx card do you have ? Maybe some firmware is missing

 $ inxi -F
System:    Host: joel-pc Kernel: 4.5.0-desktop-4omv x86_64 (64 bit) Desktop: KDE Plasma 5.6.0
           Distro: OpenMandrivaLinux 2015.0 Einsteinium
Machine:   System: Dell product: OptiPlex GX620
           Mobo: Dell model: 0HH807 Bios: Dell v: A11 date: 11/30/2006
CPU:       Dual core Intel Pentium D (-MCP-) cache: 1024 KB 
           clock speeds: max: 2793 MHz 1: 2793 MHz 2: 2793 MHz
Graphics:  Card: Intel 82945G/GZ Integrated Graphics Controller
           Display Server: OpenMandriva X.org 118.2 drivers: intel (unloaded: fbdev,vesa)
           Resolution: 1280x1024@60.02hz
           GLX Renderer: Mesa DRI Intel 945G GLX Version: 2.1 Mesa 11.2.0-rc3
Audio:     Card Intel 82801G (ICH7 Family) AC'97 Audio Controller driver: snd_intel8x0
           Sound: Advanced Linux Sound Architecture v: k4.5.0-desktop-4omv
Network:   Card: Broadcom NetXtreme BCM5751 Gigabit Ethernet PCI Express driver: tg3
           IF: enp2s0 state: up speed: 1000 Mbps duplex: full mac: 00:13:72:a7:a7:4f
Drives:    HDD Total Size: 80.0GB (10.7% used) ID-1: /dev/sda model: SAMSUNG_HD080HJ size: 80.0GB
Partition: ID-1: / size: 38G used: 5.8G (17%) fs: ext4 dev: /dev/sda1
           ID-2: swap-1 size: 2.46GB used: 0.00GB (0%) fs: swap dev: /dev/sda3
RAID:      No RAID devices: /proc/mdstat, md_mod kernel module present
Sensors:   None detected - is lm-sensors installed and configured?
Info:      Processes: 193 Uptime: 39 min Memory: 1139.7/3002.2MB Client: Shell (bash) inxi: 2.2.27

## Comment 12493
Date: 2016-03-30 22:11:32 +0000
From: @benbullard79

Created attachment 1102
/usr/share/plymouth/themes/OpenMandriva/

Seems like progress dots are the OpenMandriva default plymouth theme.

> Attached file: Screenshot_20160330_151006.png (image/png, 248129 bytes)
> Description:   /usr/share/plymouth/themes/OpenMandriva/

## Comment 12494
Date: 2016-03-30 22:18:31 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Hi Ben,
plymouth OpenMandriva is a very difficult splash to see .....
I'll tell you .... black background, openmandriva logo and 5/6 white and blue dots cycling

## Comment 12495
Date: 2016-03-30 22:30:30 +0000
From: @benbullard79

(In reply to luca from comment #11)
> Hi Ben,
> plymouth OpenMandriva is a very difficult splash to see .....
> I'll tell you .... black background, openmandriva logo and 5/6 white and
> blue dots cycling

Oh, OK. So that does show sometimes like when you reboot but tends to not show instead showing the 3 ugly squares when booting into. Will try 'plymouth-set-default-theme -R OpenMandriva' and see if it changes anything.

## Comment 12513
Date: 2016-03-31 14:17:24 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

with latest kernel(6) plymouth not even start

## Comment 12520
Date: 2016-03-31 20:06:41 +0000
From: @tpgxyz

I bet this is connected with https://issues.openmandriva.org/show_bug.cgi?id=1583

New grub2 should fix this issue.

## Comment 12524
Date: 2016-03-31 21:36:23 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Yes, now a not working plymouth is back ;)

## Comment 12527
Date: 2016-03-31 22:38:24 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

 first I find plymouth very annoying to test :)

what I see now:

I have 2 kernels 04 and 07

when I switch from one to other , Plymouth works only if after the second boot on the same (hope I am clear !)

and sometimes when it works I see at the end schedule someting or network something

sometimes it works fine

and it works always when you quit

In brief it's a mess

## Comment 12569
Date: 2016-04-03 12:00:34 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to Joel Tesson from comment #16)
>  first I find plymouth very annoying to test :)
> 
> what I see now:
> 
> I have 2 kernels 04 and 07
> 
> when I switch from one to other , Plymouth works only if after the second
> boot on the same (hope I am clear !)
> 
> and sometimes when it works I see at the end schedule someting or network
> something
> 
> sometimes it works fine
> 
> and it works always when you quit
> 
> 
> In brief it's a mess

But I still continue to test

Maybe what I will say is silly or stupid, but every time I add manually at the command line "plymouth.enable=1" I have plymouth working  whatever the chosen kernel

## Comment 12570
Date: 2016-04-03 12:09:48 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

strange as plymouth is enabled by default.

probably this chage something in boot time/order sequence

ps: in vbox no changes

## Comment 12571
Date: 2016-04-03 12:27:21 +0000
From: Joel Tesson &lt;<jtsn@free.fr>&gt;

(In reply to luca from comment #18)
 
> ps: in vbox no changes

if someone could check on a real installation would be fine

## Comment 12592
Date: 2016-04-05 16:50:25 +0000
From: @tapwag

Setting to NEEDINFO as this requires further feedback.

## Comment 12595
Date: 2016-04-05 16:57:06 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

TPG says upstream are not able to solve these problems.

so whar are we talking about?

## Comment 12596
Date: 2016-04-05 16:57:25 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

whar=what

## Comment 12643
Date: 2016-04-07 16:01:34 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

I have tested mageia 6 dev1 and plymouth 0.9.2 works.

So were are they wrong?

## Comment 12872
Date: 2016-04-16 20:28:03 +0000
From: @benbullard79

Once in for all I have 3 computers with OM Lx 3 installed and fully updated and plymouth does not work consistently on any of them. It shows properly when system is shutting down but does not show at all when system is starting.

Note I'm talking about plymouth not working on hardware on 3 computers.

## Comment 12873
Date: 2016-04-16 20:31:20 +0000
From: @benbullard79

To be more clear. Plymouth does not work consistently on OM Lx 3 .iso's on any of 3 computers and hasn't for a long time. ALSO plymouth does not work on installed systems on all 3 computers.

## Comment 12888
Date: 2016-04-17 17:08:16 +0000
From: @jclvanier

Recently, I have installed build 130 on an old laptop (Packard-Bell Butterfly S-FU-OO8)
Plymouth works fine before and after updates.

$ inxi -v 1
System:    Host: jcl-PB Kernel: 4.5.1-desktop-10omv x86_64 (64 bit) Desktop: KDE Plasma 5.6.2
           Distro: OpenMandrivaLinux 2015.0 Einsteinium
CPU:       Dual core Intel U4100 (-MCP-) speed/max: 1200/1300 MHz
Graphics:  Card-1: Intel Mobile 4 Series Integrated Graphics Controller
           Card-2: Advanced Micro Devices [AMD/ATI] RV710/M92 [Mobility Radeon HD 4330/4350/4550]
           Display Server: OpenMandriva X.org 118.3 driver: intel Resolution: 1366x768@59.97hz
           GLX Renderer: Mesa DRI Mobile Intel GM45 Express GLX Version: 2.1 Mesa 11.2.0

The graphic card used is Card-1.

## Comment 12889
Date: 2016-04-17 17:16:45 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

@jcl
if devs read only your post never will fix this issue ;)

so I'm asking you have tried in vbox? or with a nvidia card? or with a radeon?

i tell you more. sometime also in vbox it works

we need a stable situation

## Comment 12894
Date: 2016-04-17 17:53:47 +0000
From: @benbullard79

(In reply to luca from comment #27)
> @jcl
> if devs read only your post never will fix this issue ;)
> 
> so I'm asking you have tried in vbox? or with a nvidia card? or with a
> radeon?
> 
> i tell you more. sometime also in vbox it works
> 
> we need a stable situation

FWIW:The computers mentioned in Comments # 24 and # 25 all have Intel graphics and Intel i5-CPU's.

1. Intel i5-4590 with Intel® HD Graphics 4600 
   [Desktop on ASUS H97M-E motherboard]

2. Intel i5-3337 Intel® HD Graphics 4000 [ASUS X550C notebook]

3. Intel i5-2450M Intel® HD Graphics 3000 [ASUS K53E notebook]

Don't think CPU is relevant to this issue but Graphics are. All use i915 driver.

In my opinion working sometimes or working occasionally isn't working correctly.

Summary: Plymouth not working is clearly not a VirtualBox, nVidia, or Radeon issue. It is simply broken.

## Comment 12896
Date: 2016-04-17 18:05:48 +0000
From: @jclvanier

I tried on a a physical machine (PB Butterfly S).
The card used is the intel IGC. I cannot make the radeon one working.
But even if I use the radeon wich doesn't let X starting, plymouth works fine each time I start the computer.
I told because, imo, the problem is not so direct as Ben says.
I have not changed anything in the installed kernel command line.

## Comment 12900
Date: 2016-04-17 18:35:02 +0000
From: @benbullard79

(In reply to jcl vanier from comment #29)
> I tried on a a physical machine (PB Butterfly S).
> The card used is the intel IGC. I cannot make the radeon one working.
> But even if I use the radeon wich doesn't let X starting, plymouth works
> fine each time I start the computer.
> I told because, imo, the problem is not so direct as Ben says.
> I have not changed anything in the installed kernel command line.

Hmmm... I thought it was established that Plymouth does work on some hardware, works occasionally on other hardware, and doesn't work on some hardware. If so doesn't this indicate a problem that needs to be fixed?

## Comment 12902
Date: 2016-04-17 19:07:57 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

@jcl, you a lucky guy. I have no working plymouth since oma 2013.0

@Ben, You are right. This is one of the most difficult to resolve because is not manifested in the same way for all. 
But that the problem exists is undeniable.

So please, if your plymouth works be happy. 
There is no reason to file a bugreport. 
Avoid further confuse developers.

## Comment 13274
Date: 2016-05-27 00:19:52 +0000
From: @berolinux

Even more confusing: It works for me 99% of the time on both my desktop (Radeon) and my laptop (Intel). But it fails randomly, the only pattern I've seen to it is that usually it fails after a kernel update but then goes back to working (even though I'm not doing anything fancy like rebuilding initrds or so).

## Comment 13280
Date: 2016-05-27 06:46:31 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

100% tail in vbox even with latest build.
100% success with plymouth downgrade

## Comment 13313
Date: 2016-05-31 15:24:02 +0000
From: @tpgxyz

Old pre 0.9.x plymouth does not support systemd and dracut so downgrading is not an option.

Please try plymouth-0.9.3-0.20160527.8

## Comment 13316
Date: 2016-05-31 15:26:59 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to Tomasz Paweł Gajc from comment #34)
> 
> Please try plymouth-0.9.3-0.20160527.8

Please try it by yourself.

## Comment 13333
Date: 2016-05-31 16:24:56 +0000
From: @tpgxyz

(In reply to luca from comment #35)
> (In reply to Tomasz Paweł Gajc from comment #34)
> > 
> > Please try plymouth-0.9.3-0.20160527.8
> 
> Please try it by yourself.

Sure and will do. Btw thanks for your support.

## Comment 13337
Date: 2016-05-31 16:47:39 +0000
From: @jclvanier

Same behaviour in vbox with plymouth-0.9.3-0.20160527.8

## Comment 13339
Date: 2016-05-31 16:50:57 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Are you kidding? 
Are years that I try plymouth.
Now is the time for someone else.
And then I do not think a downgrade is so problematic
how does Rosa with systemd and dracut?
And why It works like a charm in my cooker?

## Comment 13447
Date: 2016-06-06 23:36:58 +0000
From: @tpgxyz

*** Bug #1289 has been marked as a duplicate of this bug. ***

## Comment 13449
Date: 2016-06-06 23:44:48 +0000
From: @tpgxyz

Plymouth does not show when running in Virtualbox environment. End of story.

Even Debain people does not want to waste time on this as it is clear VirtualBox issue with not supporting DRM well.
Plymouth < 0.9 was supporting KMS while 0.9 and newer versions rely on DRM.

https://forums.virtualbox.org/viewtopic.php?f=3&t=69765
https://www.virtualbox.org/ticket/14503

For me this is upstream issue and please let's get over it.

## Comment 13467
Date: 2016-06-07 07:19:26 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

ask mageia devs how they do it working in mageia 6(vers.0.9.2)

## Comment 13482
Date: 2016-06-07 09:59:34 +0000
From: @tpgxyz

(In reply to luca from comment #41)
> ask mageia devs how they do it working in mageia 6(vers.0.9.2)

Guess what, they synced stuff from our plymouth 4 months ago and thez are not have enabled DRM as a default renderer.

## Comment 13483
Date: 2016-06-07 10:04:52 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

and why in omv vbox sometimes it works and sometimes it doesn't?

## Comment 13500
Date: 2016-06-07 15:55:25 +0000
From: @tpgxyz

(In reply to luca from comment #43)
> and why in omv vbox sometimes it works and sometimes it doesn't?

Dunno, ask virtualbox developers who are trying for couple of years to support DRM.

## Comment 13505
Date: 2016-06-07 17:00:40 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

does it work with nvidia driver?

## Comment 13511
Date: 2016-06-07 20:14:33 +0000
From: @tpgxyz

I'm not using nvidia proprietary driver. BTW looks like newer versions of nvidia proprietary drivers supports DRM, would be nice if some volunteer could update our nvidia driver.

## Comment 13656
Date: 2016-06-11 09:37:25 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

and can you tell me why in live mode, if I switch to VT

# plymouthd
# plymouth --show-splash 

works?

## Comment 13657
Date: 2016-06-11 11:01:07 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

and adding "vboxvideo" as dracut driver plymouth works even in vbox

## Comment 13658
Date: 2016-06-11 11:41:03 +0000
From: @rugyada

Created attachment 1204
plymouth \o/

(In reply to luca from comment #48)
> and adding "vboxvideo" as dracut driver plymouth works even in vbox

Confirming !!!

> Attached file: omlx3_plymouthOMG.png (image/png, 13269 bytes)
> Description:   plymouth \o/

## Comment 13762
Date: 2016-06-16 15:36:04 +0000
From: @tpgxyz

(In reply to rugyada from comment #49)
> Created attachment 1204 [details]
> plymouth \o/
> 
> (In reply to luca from comment #48)
> > and adding "vboxvideo" as dracut driver plymouth works even in vbox
> 
> Confirming !!!

Will add x11-driver-video-vboxvideo on ISO.

## Comment 13766
Date: 2016-06-16 15:45:48 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

it is not x11-driver-video-vboxvideo needed, but vboxvideo kernel driver in vbox-additions.

I don't think are the same thing and has to be added to dracut.

but additions are not free....will be your answer ;)

## Comment 13767
Date: 2016-06-16 15:53:30 +0000
From: @tpgxyz

(In reply to luca from comment #51)
> it is not x11-driver-video-vboxvideo needed, but vboxvideo kernel driver in
> vbox-additions.
> 

Adding x11-driver-video-vboxvideo which is free will not hurt.

> I don't think are the same thing and has to be added to dracut.
> 
> but additions are not free....will be your answer ;)

Yes we are producing FREE iso, and doubt that TC will change it.
Anyways adding kernel vboxdriver means whole virtualbox and its deps are going to be pulled on ISO. So pull vbox on ISO so plymouth can display shiny animations when that ISO is booting on Vbox. Feel the recurency.

## Comment 13768
Date: 2016-06-16 16:00:53 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

why whole vbox?
only additions are needed

## Comment 13770
Date: 2016-06-16 16:12:07 +0000
From: @tpgxyz

(In reply to luca from comment #53)
> why whole vbox?
> only additions are needed

I forgot to mention about devel stuff like kernel-devel, gcc, glibc-devel and many more just to build one module to get a shiny plymouth animation inside vbox.

## Comment 13772
Date: 2016-06-16 16:17:01 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

I haven't asked nothing like that.

I only tell you that x11-driver-video-vboxvideo is probably useless, simply this.

Imho, a shiny plymouth animation inside vbox, for new users, is more important than clang.

## Comment 13773
Date: 2016-06-16 17:10:58 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to Tomasz Paweł Gajc from comment #50)

> Will add x11-driver-video-vboxvideo on ISO.

Already present in ISO

## Comment 13778
Date: 2016-06-16 20:16:32 +0000
From: @tpgxyz

(In reply to luca from comment #55)

> Imho, a shiny plymouth animation inside vbox, for new users, is more
> important than clang.

vboxvideo DRM module needs to be built first.
Adding non-free software and tons of MB of devel packages to build vboxvideo kernel module sounds crazy. Not to mention that dkms builds external kernel modules on boot.

So even if this craziness ever happen, there will be no plymouth on first vbox boot, because you have to build this module first.

Again. Convince VirtualBox developers they must focus on fixing DRM support in their Virtualbox, so you could have a shiny plymouth animation.

## Comment 13781
Date: 2016-06-16 20:51:14 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

maybe you don't understand, but I have patience.
I do not care for a shiny plymouth animation for me.
plymouth is the first thing I disable.

I'm afraid that a new user, after a not working plymouth and a calamares bigger than vbox windows change distro.

For the tons of MB of devel packages a static module is always possible.

you are not obliged to respond, we are only loosing time.

## Comment 13938
Date: 2016-06-22 15:34:24 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

FIXED!

with vboxvideo module embeded in new kernel(4.6.2-2) plymouth is working even in vobox.

now I can change distro ;)

## Comment 13939
Date: 2016-06-22 15:37:59 +0000
From: @berolinux

Except it seems to cause sddm to not start anymore, so we may have to revert it.

## Comment 13940
Date: 2016-06-22 15:46:18 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

I have installed 371(in multi-user mode) and sddm start with no problem.

## Comment 13941
Date: 2016-06-22 15:51:44 +0000
From: @berolinux

Interesting... Colin says it crashes. Maybe he is actually running into bug #1706 (which would prove that was never the Radeon drivers in the first place).

Let's kill libevent in the next build...

Is the window size ok for you as well?

## Comment 13942
Date: 2016-06-22 16:01:54 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

Bero, 
I have installed booting in multi-user mode, because in live mode doesn't start.
Than, from root prompt, startx + calamares.
Startx, in this way, open a minimal Xsession 800x600 not enought for calamares form.

But, once installed, it works as aspected: running plymouth and sddm ok.

## Comment 13943
Date: 2016-06-22 16:10:33 +0000
From: @rugyada

Confirming everything from Luca.
371 installed in multi-user mode, plymouth ok, sddm ok, system running ok.

## Comment 13944
Date: 2016-06-22 16:16:26 +0000
From: @rugyada

(In reply to Bernhard Rosenkränzer from comment #62)
> 
> Is the window size ok for you as well?

I have noticed that something changed btw.
Even if not starting nor doing anything (just black screen), window became larger. Much more than before, so I guess it could be ok once ISO will properly work as expected.

## Comment 13948
Date: 2016-06-22 17:02:07 +0000
From: luca &lt;<agoiza@gmail.com>&gt;

(In reply to Bernhard Rosenkränzer from comment #60)
> Except it seems to cause sddm to not start anymore, so we may have to revert
> it.

I tried with modeprobe.blacklist=vboxvideo as kernel parameter, but 371 doesn't start anyway.

## Comment 13971
Date: 2016-06-23 12:43:30 +0000
From: @rugyada

(In reply to rugyada from comment #65)
> (In reply to Bernhard Rosenkränzer from comment #62)
> > 
> > Is the window size ok for you as well?
> 
> I have noticed that something changed btw.
> Even if not starting nor doing anything (just black screen), window became
> larger. Much more than before, so I guess it could be ok once ISO will
> properly work as expected.

Here is what happened to me:

ISO 374 in Virtualbox

- first boot in live mode display is large
- second boot in live mode display is small. Go on and install anyway.

- first boot installed system display is small
logout (no reboot)
login
display is large.
